### ISSUE TITLE
`Style/ArgumentsForwarding` should default to `UseAnonymousForwarding: false`

### ISSUE THREAD

**Issue by dgmora** [rubocop/rubocop#13791]
Based on the example:

```ruby
# bad
def foo(**kwargs)
  bar(**kwargs)
end

# good
def foo(**)
  bar(**)
end
```

This seems reasonable. However, if at some point you want to modify `foo` and use `**`, `**` is not informative and you have to change the argument name and find out what is it supposed to be.

I don't think it's a huge issue, but given the choice I think `**<somename>` is slightly more informative and simplifies future development. I am aware it's configurable, but I wanted to ask if it'd make sense to have this as a default.

**Comment by Earlopain** [comment]
The same is also true for usage of `...` or `&`. It's the whole point of the cop and I don't think it makes sense to disable only one piece of it.

In recent rubies writing it like this is also a performance optimization: it's proven that the arguments can't be modified so it can avoid array and hash allocations.

**Comment by dgmora** [comment]
I didn't notice that this is only for 'redundant' names. I guess using a better name is the right fix (I still prefer the more communicative names, even if it's not optimized 🙂)

**Comment by dgmora** [comment]
I don't have permission to re-open, if you someone can, you can  close it as `Closed` instead of `Completed`
